packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -341,20 +336,31 @@ function shouldUpgradeListener(
 
 export function listenToNativeEvent(
   topLevelType: DOMTopLevelEventType,
-  target: EventTarget,
-  listenerMap: ElementListenerMap,
-  eventSystemFlags: EventSystemFlags,
   isCapturePhaseListener: boolean,
+  rootContainerElement: EventTarget,
+  targetElement?: Element,
```

Comment:
Let's not make many parameters optional? This makes mistakes harder to catch.

If you want to express an nullable value, you can write `targetElement: ?Element,` instead of `targetElement?: Element`. Then it would accept `null` and `undefined`, but not let you skip the arguments completely.